components/chat/OpponentChat.tsx

```
@@ -0,0 +1,56 @@
+import { colors } from "@/constants";
+import { Pressable, StyleSheet, View } from "react-native";
+import CustomText from "../CustomText";
+
+interface OpponentChatProps {}
+
+function OpponentChat({}: OpponentChatProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)